src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+    private final int index = random.nextInt(Color.values().length);
```

Comment:
currently your ColorSupplier returns the same color each time. Let's calculate index inside getRandomColor method